### PR TITLE
[CUMULUS-3201] Add new checksumType to LZARDs backup task

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes

Addresses [CUMULUS-3201: As LZARDS and CUMULUS user, I should be able to submit backup request to LZARDS with SHA512 hash so that LZARDS API can receive and process SHA512 backup requests.](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3201)

## Changes

* Add support for SHA512 in LZARDs backup task

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Restructured Earthdata Login class to be individual methods as opposed to a Class Object
   - Removed typescript no-checks and reformatted EarthdataLogin code to be more type friendly
 
+### Added
+
+- **CUMULUS-3201**
```
Comment (jennyhliu) [inline CHANGELOG.md:38]: move to 'Unreleased'

**Review by jennyhliu — CHANGES_REQUESTED** [review]
Can you add 'sha512' (sha256 as well) to lzardsBackup integration test (LzardsBackupSpec.js or LzardsBackupWorkflowSpec.js)?
Also, can we ingest granule with checksum type 'sha512' and 'sha256' in the payload?

**Comment by jennyhliu** [comment]
Suggest to update `IngestGranuleSuccessSpec` to use `IngestAndPublishGranuleWithOrca` (since orca is always enabled now), and set different checksumTypes in the input payload, and verify everything works as expected.

File: example/spec/parallel/lzardsBackup/LzardsBackupSpec.js [hunk 2]
```diff
@@ -68,93 +69,162 @@ describe('The Lzards Backup Task ', () => {
...
-                  dataType: collection.name,
-                  version: collection.version,
-                  provider,
-                  createdAt: tenMinutesAgo,
-                  files: [
-                    {
-                      fileName: 'testGranule.jpg',
-                      bucket: ingestBucket,
-                      key: `${ingestPath}/testGranule.jpg`,
-                      checksumType: 'md5',
```
[the remark this answers is not in the document]
Comment (botanical) [reply example/spec/parallel/lzardsBackup/LzardsBackupSpec.js:114]: https://github.com/nasa/cumulus/pull/3299/commits/5ed07abd06e3c4bf6f2279a1aa5c32fb0e60aff7

**Comment by botanical** [comment]
> Can you add 'sha512' (sha256 as well) to lzardsBackup integration test (LzardsBackupSpec.js or LzardsBackupWorkflowSpec.js)? Also, can we ingest granule with checksum type 'sha512' and 'sha256' in the payload? Maybe add these checksumType to the payload for IngestAndPublishGranuleWithOrca, so we can verify everything works?

What I did was update the IngestGranule payload that several of our integration tests use to include the various checksums. Let me know what you think @jennyhliu

**Review by jennyhliu — APPROVED** [review]
nice work

**Pull request merged**